### ISSUE TITLE
invalid apache config from the 'main' interface

### ISSUE THREAD

**Issue by fredj** [camptocamp/c2cgeoportal#2341]
in `apache/wsgi.conf`, the code generated for the main interface is something like:
```
RewriteRule ^/fredj/?$ /fredj/wsgi/ [PT]
RewriteRule ^/fredj/theme/(.+)$ /fredj/wsgi//theme/$1 [PT]
```

It should rather be:
```
RewriteRule ^/fredj/?$ /fredj/wsgi/main [PT]
RewriteRule ^/fredj/theme/(.+)$ /fredj/wsgi/main/theme/$1 [PT]
```

?

**Comment by sbrunner** [comment]
This is fixed?

**Comment by fredj** [comment]
fixed with #2438
